### ISSUE TITLE
What is the role of "opt.feat_bias"?

### ISSUE THREAD

**Issue by Run542968** [EricGuo5513/momask-codes#61]
Hi, thanks for this great work!

When I was reading the code, I found that each time the dataset was loaded, a new `mean.npy` and `std.npy` would be calculated based on `opt.feat_bias`. May I ask **what role** does opt.feat_bias play? And **how to determine** this hyperparameter?

Looking forward to your reply, thx!

**Comment by Murrol** [comment]
Hi, thanks for your interest. 

The mean and std are calculated during dataset loading only when training the autoencoder. The rest of the time, they are loaded from the meta files.

The feat_bias is a weight for the feature, like a weight in the loss function for each feature.

**Comment by Run542968** [comment]
Thx! I've noticed that this `feat_bias` is mainly applied to **root** and **foot**, so perhaps I can understand that the purpose of this is to **reduce AE's attention to these joints**?

**Comment by vadeli** [comment]
Hi, this was my question too.
@Run542968 Did you figure it out?

**Comment by Run542968** [comment]
Ah, I am also waiting for the author's further response. Perhaps, as he said, you can consider it as a hyperparameter. @vadeli
